docs/install-and-run.md

```diff
@@ -3,115 +3,126 @@
+in your Podfile's `post_install` section as in below.
+
+```
+post_install do |installer|
+require 'earlgrey' # Imports the EarlGrey gem for usage in your Podfile
+configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
+end
+```
 
-  Use the patten shown in the following Podfile to setup EarlGrey with the latest CocoaPods:
+4. Your complete Podfile when using EarlGrey with the latest CocoaPods should look like:
 
-  ```ruby
-  PROJECT_NAME = 'SampleApp'
-  TEST_TARGET = 'SampleAppTests'
-  SCHEME_FILE = 'SampleAppTests.xcscheme'
+```ruby
+PROJECT_NAME = 'SampleApp'
+TEST_TARGET = 'SampleAppTests'
+SCHEME_FILE = 'SampleAppTests.xcscheme'
 
-  target TEST_TARGET do
-    project PROJECT_NAME
+target TEST_TARGET do
+project PROJECT_NAME
 
-    use_frameworks! # Required for Swift Test Targets only
-    inherit! :search_paths # Required for not double-linking libraries in the app and test targets.
-    pod 'EarlGrey'
-  end
+use_frameworks! # Required for Swift Test Targets only
+inherit! :search_paths # Required for not double-linking libraries in the app and test targets.
+pod 'EarlGrey'
+end
 
-  post_install do |installer|
-    require 'earlgrey' # Imports the EarlGrey gem for usage in your Podfile
-    configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
-  end
-  ```
+post_install do |installer|
+require 'earlgrey' # Imports the EarlGrey gem for usage in your Podfile
+configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
+end
+```
 
-   * Call the `configure_for_earlgrey` method for each target / Scheme.
-   * For compatibility between different versions, please peruse [this doc](https://github.com/google/EarlGrey/tree/master/docs/versions.md).
-     To download a particular version of the gem, use `gem install earlgrey -v x.y.z`.
+5. Install the EarlGrey gem by doing `gem install earlgrey` and then do the `pod install`.
+
+Notes:
+* Call the `configure_for_earlgrey` method for each target / Scheme.
+* For compatibility between different versions, please peruse [this doc](https://github.com/google/EarlGrey/tree/master/docs/versions.md).
+To download a particular version of the gem, use `gem install earlgrey -v x.y.z`.
 
 #### Step 3: Run the pod install command
 
 After you successfully run the `pod install` command, open the generated workspace and find EarlGrey
 installed in the `Pods/` directory. The generated `Pods/` project should look similar to:
 
-  <img src="images/image02.png" width="250">
+<img src="images/image02.png" width="250">
 
 ### Carthage Installation
 
-EarlGrey supports Carthage via [the EarlGrey gem.](https://rubygems.org/gems/earlgrey)
-
 #### Step 1: Set up a test target for Carthage
 
 See [Step 1 from the `CocoaPods installation`](#step-1-set-up-a-test-target) detailed above.
 
 #### Step 2: Configure carthage
 
-Install Carthage
+1. Install Carthage
 
 `brew update; brew install carthage`
```

Comment:
I'd have to surround all in a ``` to get them to look the same. These are steps to be performed rather than code blocks, so I'd assume that should be fine. Do you think it would be better to turn these into code blocks?